content/webapp/utils/extract-works-ids.ts

```diff
@@ -0,0 +1,164 @@
+import type * as prismic from '@prismicio/client';
```

Comment:
I assume this file is a copy of the one in `content-api`, I wonder how we could ensure maintenance/changes in both is ensured... This is very much not a blocker for this PR just yet another reflection of "should we have shared helpers"?

Reply:
Interestingly, if we fetched the article content from the Content API, all of this could be removed... is it worth thinking about?

Reply:
> I think we're staying away from fetching all the content from the Content API because of handling Prismic previews

Have we looked into it, as a discovery piece? Maybe there would be a way around (like creating our own preview? No idea!). As the goal for the Content API was to use it everywhere (I think), maybe it's something we should look into again?